sdk-docs/reference/flask.md

```diff
@@ -0,0 +1,53 @@
+# Integrations
+
+The flask charms produced by PaaS App Charmer support the following
+integrations:
+
+* Ingress: [k8s charm](https://charmhub.io/traefik-k8s)
```

Comment:
Do we not support nginx-ingress-integrator as well? If not, should we just mention traefik specifically rather than "Ingress"?